server/graphql/general/src/lib.rs

```diff
@@ -271,6 +272,25 @@ impl GeneralQueries {
     ) -> Result<update_user::UpdateUserNode> {
         last_successful_user_sync(ctx)
     }
+
+    pub async fn temperature_chart(
```

Comment:
I added to general, didn't quite see the need to add it's own crate (i.e. it's just one query, as oppose to group of related queries and mutations)